data/coalition/coalition missions.txt

```diff
@@ -3111,3 +3111,129 @@ mission "Coalition: Outskirts Gauger Ready"
+			choice
+				`	"What are your books about?"`
+					goto "what are her books about"
+				`	"No, I'm not interested. How has life been as an author though?"`
+					goto "life as an author sucks"
+			label "what are her books about"
+			`	"I mostly write science fiction books. Today, I'm here to promote 'Beyond the Hyperdrive' a story in which a group of humans steal a bunch of jump drives from the Quarg. If I continue past this point I'd be spoiling it. Read it for yourself, you'll like it!"`
+			choice
+				`	"I'm interested. Wait here, I'll go buy myself a copy."`
+					goto "buy her book"
+				`	"Cool. How has life been lately as an author?"`
+			label "life as an author sucks"
+			`	"Bad," says Ann. "I've never had a bestseller or even a book that surpasses 10,000 copies sold."`
+			choice
+				`	(Tell her about the Coalition.)`
+					goto "tell her about the coalition"
+				`	"Well, that's gotta hurt. I hope you find success in your life one day.`
+					decline
+			label "buy her book"
+			action
+				payment -60
+			`	You enter the bookstore, ask the employee for one of Ann's books, and purchase it for 60 credits.`
+			choice
+				`	(Now go to Ann and get it signed.)`
+					goto "get it signed"
+				`	(Leave.)`
+					decline
+			label "get it signed"
+			`	You move to the table where Ann is sitting. She looks visibly happy as she grabs your book, takes out a pen, and signs it. "Thanks for buying one of my books. I don't get much sales and people like you mean the world to me."`
+			choice
+				`	"I appreciate it. How has life as an author been though?"`
+					goto "life as an author sucks"
+				`	"Thanks for signing my copy. I hope you find success in your life. Goodbye."`
+					decline
+			label "tell her about the coalition"
+			`	You check your surroundings, looking around to ensure you are not being listened to. Once you confirm that your conversation will be private, you begin speaking.`
+			`	"You know, I know of a place called the Coalition. They're a group of alien races. Maybe you could go there and find inspiration for your next novel, or sell your books there? I bet they'd love to see your writings!"`
+			`	"Wait, really?" asks Ann.`
+			`	"Yes", you say.`
+			`	"But they wouldn't speak English. How are they going to read my books?"`
+			choice
+				`	"They have a translation device."`
+				`	"I'm sure they can learn the language."`
+			`	"That's reassuring... but, how am I even going to get there?"`
+			choice
+				`	"I'll take you there."`
+			`	"Do I have to pay anything? As a struggling author, I really can't pay you."`
+			choice
+				`	"It's okay. I'm willing to make an exception for you."`
+			`	"Thank you so much! Just wait for a while, I need to finish my book signing event. It should be roughly 45 minutes before it's over, and then we can go to whatever this Coalition thing is," says Ann.`
+			`	You reply. "Good luck. By the way, we're going to <destination> in Coalition space.`
```

Comment:
Might want to warn her it's giant beetles, giant spiders and centaurs. That, or add something of a reaction from her when she first sees the Coalition inhabitants. Or have both.

Reply:
a3ff8fd06dc73ecb5ca957f617fe856807a18ecd